examples/exex/op-bridge/src/main.rs

```diff
@@ -18,13 +18,13 @@ use crate::L1StandardBridge::{ETHBridgeFinalized, ETHBridgeInitiated, L1Standard
 
 /// An example of ExEx that listens to ETH bridging events from OP Stack chains
 /// and stores deposits and withdrawals in a SQLite database.
-struct OPBridgeExEx<Node: FullNodeTypes> {
-    ctx: ExExContext<Node>,
+struct OPBridgeExEx<Node: FullNodeTypes, Pool: TransactionPool> {
```

Comment:
always preferable to not set trait bounds on the generic on the type definition, but first on the impl. it makes it much easier to mock in tests, and makes type defs smaller, that may incorporate this type.

Reply:
not possible here